fdmi/module.c

```diff
@@ -1,6 +1,6 @@
 /* -*- C -*- */
 /*
- * Copyright (c) 2017-2020 Seagate Technology LLC and/or its Affiliates
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

Comment:
I am sorry if I misled you.
For newly created files, you may write:
```
* Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

For modified files, you may update the year to this year, e.g.
```
* Copyright (c) 2017-2021 Seagate Technology LLC and/or its Affiliates
```